server/data/mappers/timelineMapper.test.ts

```diff
@@ -1,13 +1,14 @@
-import moment from 'moment'
+import { parseISO } from 'date-fns'
```

Comment:
Probably easier to read the comments on the next file (the implementation) to understand the rational for these changes